real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/serial/RBSerialMessage.java

```diff
@@ -1,45 +1,79 @@
 package com.roboclub.robobuggy.serial;
 
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.roboclub.robobuggy.main.RobobuggyConfigFile;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Map;
+import java.util.Scanner;
+
 /**
  * Class representing a robobuggy serial message
  */
-public class RBSerialMessage 
+public class RBSerialMessage
 {
 
-	public static final byte ENC_TICKS_SINCE_LAST = (byte)0;
-	public static final byte ENC_TICK_SINCE_RESET = (byte)1;
-	public static final byte ENC_MS_SINCE_RESET = (byte)2;
-
-	public static final byte RBSM_MID_MEGA_COMMAND = (byte)19;
-	public static final byte STEERING = (byte)20;
-	public static final byte BRAKE = (byte)21;
-	public static final byte AUTO = (byte)22;
-	public static final byte BATTERY = (byte)23;
-	public static final byte RBSM_MID_MEGA_STEER_FEEDBACK = (byte)24; //potentiometer 
-	public static final byte RBSM_MID_MEGA_STEER_ANGLE = (byte)20; //Arduino to servo
-	public static final byte FP_HASH = (byte)30;
-	
-	public static final byte LIGHTING_ID = (byte)50;
-	
-	public static final byte RBSM_MID_RESERVED = (byte)252;
-	public static final byte RBSM_MID_ERROR = (byte)254;
-	public static final byte RBSM_MID_DEVICE_ID = (byte)255;
```

Comment:
Oh nvm, I see it defined above.  I'm not 100% sure if low level will convert -1 to 255 and -2 to 254.  I'll need to test and verify that.